modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/servlet/taglib/ui/SharingEntryDropdownItemContributorRegistryUtil.java

```diff
@@ -20,46 +20,40 @@
 import java.util.Collections;
 import java.util.List;
 
+import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
-import org.osgi.service.component.annotations.Reference;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Adolfo Pérez
  */
-@Component(service = SharingEntryDropdownItemContributorRegistry.class)
-public class SharingEntryDropdownItemContributorRegistry {
+public class SharingEntryDropdownItemContributorRegistryUtil {
 
-	public SharingEntryDropdownItemContributor
+	public static SharingEntryDropdownItemContributor
```

Comment:
Hey @RenanUmV, change the method to receive the `String className` instead. With that, you can remove the `ClassNameLocalService` dependency. Thx